.github/workflows/android-cd.yml

```
@@ -0,0 +1,43 @@
+              run: chmod +x ./gradlew
+            - name:  Build Debug APK only
+              run: ./gradlew assembleDebug
+
+            - name: Upload Artifact to Firebase App Distribution
+              uses: wzieba/Firebase-Distribution-Github-Action@v1
+              with:
+                  appId: ${{ secrets.FIREBASE_APP_ID }}
+                  serviceCredentialsFileContent: ${{ secrets.CREDENTIAL_FILE_CONTENT }}
+                  groups: "testersteam"
+                  file: app/build/outputs/apk/debug/app-debug.apk
+                  releaseNotes: ${{ inputs.release_notes }}
```

actionlint (1.7.9)

property "release_notes" is not defined in object type {}

(expression)